HA.md

```diff
@@ -0,0 +1,21 @@
+# High Availability (HA)
+## Introduction
+When we create kubernetes cluster using kops we are able to create a multiple K8S masters and by doing so we can assure that in a case of 
```

Comment:
> When we create kubernetes cluster using kops we are able to create a multiple K8S masters and by doing so we can assure that in a case of a master instance failure our K8S functionality won't damaged.

When we create kubernetes cluster using kops we are able to create multiple K8S masters and by doing so we can provide a HA control plane.

^ something like that??